musl and glibc use different string values for the ECHILD errno.

diff -upr gawk-5.1.0.orig/test/testext.ok gawk-5.1.0/test/testext.ok
--- gawk-5.1.0.orig/test/testext.ok	2021-02-06 12:39:05.990158877 +0100
+++ gawk-5.1.0/test/testext.ok	2021-02-06 12:39:24.080236494 +0100
@@ -21,7 +21,7 @@ var_test: sym_update of ARGC failed - co
 var_test: sym_update("testvar") succeeded
 var_test() returned 1, test_var = 42
 
-test_errno() returned 1, ERRNO = No child processes
+test_errno() returned 1, ERRNO = No child process
 
 fubar = 9
 rumpus = -5
